### DISCUSSION TITLE
Transaction stuck in the txpool on OP stack replica

### DISCUSSION THREAD

**Discussion by parkerjou** [ethereum-optimism/developers#445] [Node Operators]
**Bug Description**
I'm running a custom OP stack rollup and trying to sync up a replica node locally to the upstream rollup. The replica node appears to be syncing to tip and queries to the replica node show the correct result. I'm running into issues trying to submit transactions to the replica node (to be broadcast upstream). I submit the transaction to the node but it seems to be stuck in the tx pool.

The --rollup.sequencerhttp flag is specified to the rpc url of the upstream node. The logs state "Transaction submitted".

curl http://localhost:8545 \
-X POST \
-H "Content-Type: application/json" \
--data '{"method":"txpool_content","params":[],"id":1,"jsonrpc":"2.0"}'  

Output:
```json{
    "jsonrpc": "2.0",
    "id": 1,
    "result": {
        "pending": {
            "0xA8427088AB90Da02E40deFF559510bCB0f1CD11b": {
                "9": {
                    "blockHash": null,
                    "blockNumber": null,
                    "from": "0xa8427088ab90da02e40deff559510bcb0f1cd11b",
                    "gas": "0xc350",
                    "gasPrice": "0x746a528864",
                    "maxFeePerGas": "0x746a528864",
                    "maxPriorityFeePerGas": "0x746a528800",
                    "hash": "0x1f41e92ec13fe5db8369465bd4963300293ceba0571e107ecf0b6c45b78a1fbd",
                    "input": "0x",
                    "nonce": "0x9",
                    "to": "0x61109b4b54fcaf2c58f056551cf9c9b72404b689",
                    "transactionIndex": null,
                    "value": "0x2386f26fc10000",
                    "type": "0x2",
                    "accessList": [],
                    "chainId": "0x34816e",
                    "v": "0x0",
                    "r": "0x4923c3187e96b1cc400c5e89cbf31d810164eeaca83ff3fd2a8720ac2e2f6e8",
                    "s": "0x4e7cfd8a7c6629ccd4bf85042ae5ac768169114148abf428d3f47c894fefe8d8",
                    "yParity": "0x0"
                }
            }
        },
        "queued": {}
    }
}
```

**Steps to Reproduce**

1. Happy to provide more steps for reproduction if nothing rings a bell with the initial description.

**Expected behavior**
I expect the transaction to be broadcast upstream and to be processed by the upstream node.

**Environment Information:**
 - Operating System: Ubuntu 20.04
 - Package Version (or commit hash): op-geth:v1.101308.2

**Configurations:**

exec geth \
	--datadir="$GETH_DATA_DIR" \
	--verbosity="$VERBOSITY" \
	--http \
	--http.corsdomain="*" \
	--http.vhosts="*" \
	--http.addr=0.0.0.0 \
	--http.port="$RPC_PORT" \
	--http.api=web3,debug,eth,txpool,net,engine \
	--ws \
	--ws.addr=0.0.0.0 \
	--ws.port="$WS_PORT" \
	--ws.origins="*" \
	--ws.api=debug,eth,txpool,net,engine \
	--syncmode=full \
	--nodiscover \
	--maxpeers=1 \
	--networkid=$CHAIN_ID \
	--unlock=$BLOCK_SIGNER_ADDRESS \
	--mine \
	--miner.etherbase=$BLOCK_SIGNER_ADDRESS \
	--password="$GETH_DATA_DIR"/password \
	--allow-insecure-unlock \
	--authrpc.addr="0.0.0.0" \
	--authrpc.port="8551" \
	--authrpc.vhosts="*" \
	--authrpc.jwtsecret=/config/jwt-secret.txt \
	--gcmode=archive \
	--metrics \
	--metrics.addr=0.0.0.0 \
	--metrics.port=6060 \
	--rollup.sequencerhttp=$UPSTREAM_RPC_URL
	"$@"

**Logs:**
Logs and/or error messages that help illustrate the issue.

**Additional context**

---

⚠️ **Notice:** Issues that do not include the following sections will be subject to closure:
- Bug Description
- Steps to Reproduce
- Environment Information

Please ensure all required sections are filled out accurately to expedite the debugging process and improve issue resolution efficiency.

**Comment by sbvegan** [comment]
A few things you can try to look into:

You should first try to run the latest releases, so for [v1.101315.1 - op-geth](https://github.com/ethereum-optimism/op-geth/releases/tag/v1.101315.1) and [op-node v1.7.6](https://github.com/ethereum-optimism/optimism/releases/tag/v1.7.6). 

The following configurations may result in peering issues with the sequencer node and other replica nodes. I imagine the `--rollup.sequencerhttp=$UPSTREAM_RPC_URL` should handle forwarding this transaction, but given the transaction is not being processed; I don't think its making it to the sequencer.

```
--nodiscover
--maxpeers=1
```

Some things that might be helpful to know:

- What network is this on?
- Do you have any visibility into the sequencer node? 
- Can you share you op-node configuration too?

**Comment by opfocus** [comment]
```
--mine
--miner.etherbase=$BLOCK_SIGNER_ADDRESS
--password="$GETH_DATA_DIR"/password 
```    why you use mine here?
